application/src/main/java/org/opentripplanner/graph_builder/module/osm/VertexGenerator.java

```diff
@@ -95,6 +106,24 @@ IntersectionVertex getVertexForOsmNode(OsmNode node, OsmWithTags way) {
         iv = bv;
       }
 
+      if (includeOsmSubwayEntrances && node.isSubwayEntrance()) {
+        String ref = node.getTag("ref");
+
+        boolean accessible = node.isTag("wheelchair", "yes");
+
+        FeedScopedId id = new FeedScopedId("osm", Long.toString(nid));
+        Entrance entrance = Entrance
+          .of(id)
+          .withCoordinate(new WgsCoordinate(coordinate))
+          .withCode(ref)
+          .withWheelchairAccessibility(
+            accessible ? Accessibility.POSSIBLE : Accessibility.NOT_POSSIBLE
```

Comment:
Should we also check if the tag is defined? If not, the value should be `NO_INFORMATION`.